src/garage/torch/modules/multi_headed_mlp_module.py

```diff
@@ -0,0 +1,125 @@
+import torch.nn as nn
+
+
+class MultiHeadedMLPModule(nn.Module):
+    ERROR_MESSAGE = '{} should be either an integer or a collection of length n_heads ({}), but {} provided.'
```

Comment:
this is not necessary. please just put it right in the code.

Reply:
fixed